### PR TITLE
fixing the newlatter form and sticky navigation

### PR DESCRIPTION
**Pull request by bruxx-6243**

### CODE REVIEW

File: src/components/GetInTouch.astro [hunk 1]
```diff
@@ -15,7 +15,7 @@ import Button from "./Button.astro";
       >
         Type your email down below and be young wild generation
       </p>
-      <div>
```
Comment (bruxx-6243) [inline src/components/GetInTouch.astro:18]: Good

File: src/components/GetInTouch.astro [hunk 2]
```diff
@@ -15,7 +15,7 @@ import Button from "./Button.astro";
...
+      <div class="relative">
```
Comment (bruxx6243) [inline src/components/GetInTouch.astro:18]: I really don't know I should write review always

File: src/components/GetInTouch.astro [hunk 3]
```diff
@@ -29,12 +29,16 @@ import Button from "./Button.astro";
           <button
             data-btn-send
             type="submit"
-            class="text-base lg:text-lg text-white bg-black px-4 py-2 rounded-[7px]"
+            class="text-base lg:text-lg text-white bg-black flex items-center justify-center w-[70px] h-[45px] rounded-lg"
```
Comment (bruxx6243) [inline src/components/GetInTouch.astro:32]: Good to go

**Review by bruxx6243 — COMMENTED** [review]
This PR is approved

**Pull request merged**